lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -124,9 +137,34 @@ _InsertionLocation _insertionLocationForPackageImport(
         trailingNewlineCount: 2);
   }
 
+  imports.forEach((importDirective) {
```

Comment:
It's probably preferable to use a regular for-loop for this. They're easier to debug in Dart. Or you could write it as two "any" loops. 

hasPackageImports = imports.any((importDirective) => ...
hasRelativeImports = imports.any(...

Reply:
Updated with `for` loop